shiny/http_server.py

```diff
@@ -0,0 +1,13 @@
+from flask import Flask
+
+
+app = Flask(__name__)
+
+
+@app.route('/v1/do-nothing/<unicorn_name>', methods=['POST'])
+def do_nothing(unicorn_name):
+    return ''
```

Comment:
Do you want me to still do this? It'll take some massaging of the path, but it's possible.